src/applications/healthcare/questionnaire/tests/e2e/01.questionnaire.demographics.cypress.spec.js

```diff
@@ -0,0 +1,71 @@
+const Timeouts = require('platform/testing/e2e/timeouts.js');
```

Comment:
With Cypress's automatic timeouts, there's generally [no need to use the timeouts from our Nightwatch helpers](https://github.com/department-of-veterans-affairs/va.gov-team/blob/master/platform/testing/end-to-end/cypress-migration-guide.md#automatic-waiting).